kata-webhook/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+FROM golang:1.10 AS builder
+
+WORKDIR /go/src/kata-pod-annotate
+
+COPY . ./
+RUN go get ./... && CGO_ENABLED=0 go build -o /go/bin/kata-pod-annotate
+
+FROM alpine:3.7
```

Comment:
Is there any reason this couldn't be specified as `alpine:latest` to avoid what will eventually become an obsolete version?